source/services/encryption/EncryptionService.stories.tsx

```diff
@@ -35,13 +45,74 @@ const runTerminalDemo = () => {
   reactNativeAesDemo();
 };
 
+const printPublicKeyResult = (user, form) => {
+  console.log(
+    `Updated user ${user.personalNumber} public key: ${
+      form.encryption.publicKey.publicKeys[user.personalNumber]
+    }`
+  );
+};
+
+const printSymmetricKeyResult = async (user, form) => {
+  console.log(`User ${user.personalNumber} symmetric key ${await getStoredSymmetricKey(form)}`);
+};
+
+const testSymmetricKeySetup = async () => {
+  const mainApplicantYlva = {
+    personalNumber: '196912191118',
+  };
+  const coApplicantStina = {
+    personalNumber: '198310011906',
+  };
+  const testForm = {
+    answers: { encryptedAnswers: 'This string will be encrypted' },
+    encryption: {
+      type: 'decrypted',
+      publicKey: {
+        P: 43,
+        G: 10,
+        symmetricKeyName: '196912191118:198310011906',
+        publicKeys: {
+          196912191118: undefined,
+          198310011906: undefined,
+        },
```

Comment:
This feels a bit odd, ie encryption.publicKey.publicKeys. I get the intention of the data, but could we perhaps rename the publicKey to just keys?

Or have something like:

encryption: {
  ...,
  type: 'decrypted',
  keys: [{
   type: publicKey,
   value: undefined,
   id: 19....
  },
  {
   type: publicKey,
   id: 19....,
   value: undefined
     },
  {
   type: symetricKey,
   id: '196912191118:198310011906',
   value: undefined
  },
  ]
}

Reply:
Would something like this look a better?
```json

encryption: {
      "type": "decrypted",
      "symmetricKeyName": "196912191118:198310011906",
      "Primers": {
        "P": 43,
        "G": 10,
      },
      "publicKeys": {
        "196912191118": 1234,
        "198310011906": 5678,
      },
    },

```

Reply:
I like that proposed structure (except I would rename `Primers` to `primes` instead).

Reply:
If primers refers to prime numbers, i think primes is a better naming or all out primeNumbers. But if there is a good cause due to DH docs etc to use the name primers i think we should stick to it in order to ease the learning curve (y)